Website/src/rendering/src/components/SessionInformation.tsx

```diff
@@ -47,39 +63,41 @@ const SessionInformation = (props: SessionInformationProps): JSX.Element => {
     : props.fields.Speakers.length == 1
     ? 'Speaker'
     : 'Speakers';
+
   return (
-    <section className="section">
+    <section className="section information-section session-information">
       <div className="section__content left__content">
-        <div className="grid grid-cols-1 md:grid-cols-4 gap-10">
-          <div className="col-span-1 md:col-span-1">
-            <Image field={props.fields?.Image} alt={props.fields?.Name?.value} />
+        <div className="information-grid">
+          <div className="image-col">
             <div>
-              <DateField
-                tag="h3"
-                field={props.fields.Date}
-                render={(date) =>
-                  'Date: ' +
-                  date?.toLocaleDateString('en-US', {
-                    weekday: 'short',
-                    year: 'numeric',
-                    month: 'long',
-                    day: 'numeric',
-                  })
-                }
-              />
-              <DateField
-                tag="span"
-                field={props.fields.Date}
-                render={(date) =>
-                  'Time: ' +
-                  date?.toLocaleTimeString('en-US', {
-                    hour: '2-digit',
-                    minute: '2-digit',
-                  })
-                }
-              />
+              <Image field={props.fields?.Image} alt={props.fields?.Name?.value} />
+              {props.fields.Rooms && props.fields.Rooms.length > 0 && (
+                <div className="info-text">
+                  <span>
+                    <FontAwesomeIcon className="icon" icon={faDoorOpen} />
+                  </span>
+                  <Text tag="p" field={props.fields.Rooms[0].fields.Name}></Text>
+                </div>
+              )}
+              {props.fields.Day &&
+                props.fields.Day.length > 0 &&
+                props.fields.Day.map((day, index) => (
+                  <div key={index} className="info-text">
+                    <span>
+                      <FontAwesomeIcon className="icon" icon={faCalendar} />
+                    </span>
+                    <Text tag="p" field={day.fields.Name}></Text>
+                  </div>
+                ))}
+              {props.fields.Timeslots && props.fields.Timeslots.length > 0 && (
+                <div className="info-text">
+                  <span>
+                    <FontAwesomeIcon className="icon" icon={faClock} />
+                  </span>
+                  <span>{GetSessionTime(props.fields.Timeslots)}</span>
+                </div>
+              )}
             </div>
-
             <div className="hidden md:block pt-4" title="TODO: hardcoded for now...">
```

Comment:
Could you please also move the Tailwind classes from this block and children elements and the block at the end of the function to the css file? They are the only remaining inline Tailwind in this file.

Reply:
This block still has inline Tailwind. This is not resolved.

Also, I just realized that related sessions HTML is repeating itself twice on this page. This is not really optimal and is a bad accessibility practice.

Related sessions can get really squeezed just before the medium breakpoint:
![image](https://user-images.githubusercontent.com/2000200/134970089-f895ab5e-ce16-4be1-9a78-73a9b7edc223.png)

I would vote for them to be displayed only in one location, which is below the current session description.

Reply:
https://dev.azure.com/sitecoredst/Demo/_workitems/edit/6539